benchmarks/asv.conf.json

```diff
@@ -0,0 +1,167 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+    // The name of the project being benchmarked
+    "project": "dedupe",
+    // The project's homepage
+    "project_url": "https://github.com/dedupeio/dedupe",
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "..",
+    // The Python project's subdirectory in your repo.  If missing or
+    // the empty string, the project is assumed to be located at the root
+    // of the repository.
+    // "repo_subdir": "",
+    // Customizable commands for building, installing, and
+    // uninstalling the project. See asv.conf.json documentation.
+    //
+    "install_command": [
+        "python -m pip install -r requirements.txt",
```

Comment:
what is the current path? 

in some parts of the code it seems like its "." and sometimes it seems like "./benchmarks"

Reply:
confusing, I agree. See [here](https://asv.readthedocs.io/en/stable/asv.conf.json.html#build-command-install-command-uninstall-command). It is under the repo root for both of these (I am pretty sure)